torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1445,6 +1445,8 @@ class DimDynamic(Enum):
     SIZE_LIKE_UNBACKED = 3
     # Infer the strides from stride. If size is static, strides will be static as well.
     INFER_STRIDE = 4
+    # Like SIZE_LIKE_UNBACKED, but there's a hint
+    OBLIVIOUS_SIZE = 5
```

Comment:
[if what I wrote above is false, then read on, otherwise ignore]

Let me make sure I understand:

SIZE_LIKE_UNBACKED: Expands into size-like, unbacked symbolic integer. This means we do specialize on 0/1 and we don't allow guards.

OBLIVIOUS_SIZE: Expands into size-oblivious, backed symbolic integer. This means we don't specialize on 0/1 and we do allow guards.

Is this correct? If so, OBLIVIOUS_SIZE really isn't like SIZE_LIKE_UNBACKED right? If anything it's the opposite? If this is the case, maybe it'd be more clear to rename to OBLIVIOUS_SIZE_BACKED

Reply:
(see comment above)